### ISSUE TITLE
Ability to order by Axis value

### ISSUE THREAD

**Issue by kamilzakiev** [akvelon/PowerBI-Stacked-Column-Chart#12]
Same for Bar and Column charts: would be great to have ability to sort bars/columns by axis values

**Comment by antoniograzioli** [comment]
same here, I crated some small multiples (by year/month) but I cannot sort them properly. Would be really awesome to have some sort option for the multiples

**Comment by torben87** [comment]
Agreed. You have a really cool product, but not being able to sort by axis value means that I often have to use another solution.
